scopes/workspace/workspace/workspace.ts

```diff
@@ -286,6 +287,13 @@ export class Workspace implements ComponentFactory {
     return this.consumer.getPath();
   }
 
+  /**
+   * Get the location of the bit roots folder
+   */
+  get rootComponentsPath() {
+    return path.join(this.config.rootComponentsDirectory ?? this.modulesPath, BIT_ROOTS_DIR);
```

Comment:
something looks missing to me here
in case it was configured you don't have the workspace path.
if it's not configured you will have
`/path-to-workspace/node_modules/.bit_roots`
if it's configured, you will have
`/configured-path/.bit_roots` 
without the workspace path in the beginning